test/solvers/functional/functional_test.py

```diff
@@ -280,11 +280,9 @@ def test_functional_sum(space):
                             func2.gradient(x).inner(p)),
                         places=PLACES)
 
-    # Test that prox and convex conjugate is not known
+    # Test that prox is not known
     with pytest.raises(NotImplementedError):
         (func1 + func2).proximal
-    with pytest.raises(NotImplementedError):
```

Comment:
The convex conjugate should not be known for a sum. Wouldn't you still like to check something here, like that `(func1 + func2).convex_conj` is a `FunctionalDefaultConvexConjugate` or that  `(func1 + func2).convex_conj(x)` raises `NotImplemetedError`? Or are those kind of test just redundant?